Lombiq.Tests.UI/Extensions/NavigationUITestContextExtensions.cs

```diff
@@ -189,9 +196,11 @@ await context.DoWithRetriesUntilNavigationHasOccurredOrFailAsync(
         return page;
     }
 
+    [Obsolete("Methods using Page<> classes will be removed in the next version.")]
     public static Task<OrchardCoreSetupPage> GoToSetupPageAsync(this UITestContext context, bool navigate = true) =>
         context.GoToPageAsync<OrchardCoreSetupPage>(navigate);
 
+    [Obsolete("Methods using Page<> classes will be removed in the next version.")]
```

Comment:
Point people to the new method corresponding to each method too, like "Use GoToLoginAsync instead."